<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DutyNotificationDialog</class>
 <widget class="QDialog" name="DutyNotificationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notification</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="m_textCtrl"/>
      </item>
      <item>
       <widget class="Line" name="m_line1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="m_textCtrl2"/>
      </item>
      <item>
       <widget class="Line" name="m_line2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="m_textCtrl3"/>
      </item>
      <item>
       <widget class="Line" name="m_line3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="m_textCtrl4"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DutyNotificationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DutyNotificationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
